src/__tests__/earl-loader.js

```diff
@@ -0,0 +1,45 @@
+const mockFiles = {
+	'/dir/file1.json': '{ "organisation": "Deque" }',
+	'/dir/file2.json': '{ "organisation": "Siteimprove" }',
+}
+
+describe(`earlLoader`, () => {
+	jest.mock('fs')
+
+	beforeEach(() => {
+		jest.resetModules()
+		jest.resetAllMocks()
```

Comment:
I don't think you should mock globby or fs. These tests are more reliable if you just create a dir, put a bunch of files in it, test those, and then delete the dir at the end. There's no chance of bugs in your mocks messing things up, and the performance benefit you get from mocking definitely isn't worth the effort it takes to write / maintain the mocks.